### DISCUSSION TITLE
What's the command for searching package from the registry?

### DISCUSSION THREAD

**Discussion by ianzone** [oven-sh/bun#8992] [Q&A]
I couldn't find it in docs.

**Comment by matoruru** [comment]
@ianzone 
For investigating dependencies of your package, you can use `bun pm ls --all` but if you want to search from registry, you should use https://www.npmjs.com/

**Reply by ianzone** [reply]
I mean, there should be a command like `bun search`, rather than open a browser.

**Comment by matoruru** [comment]
@ianzone I hope I could help but that kind of command doesn't exist in bun based on `bun -h`

**Comment by Woody88** [comment]
For anyone else stumbling on this, you can use `bun info <package_name>`.  You can also use `bun info <package> --json` to output package versions.
